proto/kvrpcpb.proto

```diff
@@ -861,3 +861,9 @@ message KeyRange {
     bytes start_key = 1;
     bytes end_key = 2;
 }
+
+enum ExtraOp {
+    Noop = 0;
+    ReadDeleted = 1;
+    ReadOldValue = 2;
```

Comment:
Can we have them both set in the future?

Reply:
`ReadDeleted` is included in `ReadOldValue`.

Reply:
Could you add some comments? The behavior is not obvious to me.